extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -805,6 +813,55 @@ export class JupyterServerInstallation implements IJupyterServerInstallation {
 
 		return undefined;
 	}
+
+	public static getRequiredPackagesForKernel(kernelName: string): PythonPkgDetails[] {
+		let packages = [{
+			name: 'jupyter',
+			version: '1.0.0'
+		}];
+		switch (kernelName) {
+			case constants.python3DisplayName:
+				break;
+			case constants.pysparkDisplayName:
+			case constants.sparkScalaDisplayName:
+			case constants.sparkRDisplayName:
```

Comment:
Stuff like this would be nice to have in a separate config instead of inline. Don't necessarily have to do it now but something to think about in the future.

Reply:
I'll probably revisit it when I add in the checkboxes for other packages to include in the install. That will be the next feature after this gets checked in.